addon/mixins/nested-relations.js

```diff
@@ -51,6 +51,17 @@ const attributesFor = function(record) {
     }
   });
 
+  if (relationshipsHash !== undefined) {
+    iterateRelations(record, relationshipsHash, (name, kind, related, subRelations, manyToManyDeleted) => {
+      let metadata = record.relationshipFor(name);
+
+      if (metadata.options && metadata.options.serializeIdsInAttributes) {
+        let keyName = `${serializer.keyForAttribute(name)}_ids`;
```

Comment:
This only handles `hasMany` relationships correctly. It doesn't work with `belongsTo`.

Reply:
Yeah it is [currently] intentionally specific to hasMany relationships